.github/CODEOWNERS

```diff
@@ -1,8 +1,4 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
 
-# For syntax help see:
-# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
-
-* @googleapis/actools-python @googleapis/yoshi-python
-*.yaml @googleapis/yoshi-python @googleapis/actools-python
+*                         @googleapis/python-core-client-libraries
```

Comment:
This team does not have write access to the repository, so we're getting a GitHub warning on this file.

The team settings are [here](https://github.com/orgs/googleapis/teams/python-core-client-libraries/repositories), but I don't see how to add a new repo. Maybe I don't have the needed permissions?

@parthea

Reply:
Good catch! I granted write permission to the team via the Github Setting UI (Settings->Collaborators and Teams)